pyproject.toml

```diff
@@ -1,9 +1,57 @@
 [build-system]
-requires = [
-    "setuptools >= 46.4.0",
-]
+requires = ["setuptools >= 70.0.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
```

Comment:
Does dependabot understand this format?
I know that dependabot for poetry still requires the old `[tool.poetry]` section instead of the recommended `[project]`, I wonder if the setuptools-based check is different.

Reply:
I don't think dependabot updates abstract deps anyway. Renovate probably would. Though, in general, this shouldn't be updated automatically. It's not a lock file and mustn't pin things or contain top boundaries in general. This is something that pip-tools constraints would solve, as they'd lock envs w/o the dangers of causing problems in dependency resolution for the end users.

Reply:
As mentioned earlier, I reverted the part that moved the dependencies to only focus on the static metadata here.

Reply:
Ah, I didn't notice that you marked those as dynamic.

Reply:
But still, there's no need to separate those anyway. So let's get them back into this config.

Reply:
Ack. Follow-up is good. Marking the thread as resolved.